docs/pages/desktop-access/active-directory.mdx

```diff
@@ -0,0 +1,216 @@
+- An Active Directory domain, configured for LDAPS (Teleport requires an
+  encrypted LDAP connection). Typically this means installing
+  [AD CS](https://learn.microsoft.com/en-us/windows-server/identity/ad-cs/)
+
+## Step 1/2. Install Teleport
+
+### Set up the Teleport Auth and Proxy Services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+## Step 2/2. Run the discovery wizard
+
+In your web browser, access the teleport Web UI at <ScopedBlock scope={["oss", "enterprise"]}>
+`teleport.example.com`</ScopedBlock><ScopedBlock scope={["cloud"]}>
+`mytennant.teleport.sh`</ScopedBlock>. Click on your user name at the top right
+and select **Manage Access**, Select "Desktop" from the main menu, then **NEXT**:
+
+If you already have Active Directory installed, skip to the next step. Otherwise,
+copy and paste the first command provided into a Windows PowerShell window. If
+you aren't already running AD Certificate services, copy and paste the second
+command after the first one completes and the server restarts:
+
+![Install Active Directory](../../img/desktop-access/install-active-directory.png)
+
+Once the server is restarted from one or both command executions, click **NEXT**.
+
+Copy and paste the provided command into a Windows PowerShell window to download
+and run the 'configure Active Directory' script:
+
+![Configure Active Directory](../../img/desktop-access/configure-active-directory.png)
+
+Click **NEXT**.
+
+The PowerShell script will output a Teleport configuration block. Copy this
+block to a temporary location. Click **Next**.
+
+On the Linux host you installed Teleport to run as the Desktop Access connector,
+edit `/etc/teleport.yaml` and paste the configuration provided by the output of
+the previous step. Review and edit the addition as necessarily for your environment:
+
+```yaml
+version: v3
+teleport:
+  auth_token: 2239...c5b21
+  proxy_server: teleport.example.com:443
+
+auth_service:
+  enabled: no
+ssh_service:
+  enabled: no
+proxy_service:
+  enabled: no
+
+windows_desktop_service:
+  enabled: yes
+  ldap:
+    addr:     '10.10.1.50:636'
+    domain:   'windows.teleport.example.com'
+    username: 'WIN\svc-teleport'
+    server_name: 'windows-server-hostname'
+    insecure_skip_verify: false
+    ldap_ca_cert: |
+        -----BEGIN CERTIFICATE-----
+        MIIDnzCCAoegAwIBAgIQT/UIn+MT4aZC9ix/QuiV9zANBgkqhkiG9w0BAQsFADBi
+        ...
+        31qA4dO3if7RdikD9hVbiIF9jQ==
+        -----END CERTIFICATE-----
+
+  discovery:
+    base_dn: '*'
+  labels:
+    teleport.internal/resource-id: 42d8859c-60d0-4d7f-9767-bdd66b63fce6
+```
+
+Click **Next**.
+
+Once you've saved `/etc/teleport.yaml`, start Teleport:
+
+<Tabs>
+<TabItem label="System Service">
+
+This command assumes the `root` user. Prepend `sudo` otherwise.
+
+```code
+$ systemctl start teleport.service
+```
+
+Copy the join token to a file on the instance where you will run the Windows Desktop
```

Comment:
Since we also include this snippet above (with the pre-populated information) should we delete this paragraph and the snippet below?

Reply:
Removed.